packages/react/index.ts

```diff
@@ -347,23 +348,23 @@ function reactAxe(
   _React: typeof React,
   _ReactDOM: typeof ReactDOM,
   _timeout: number,
-  conf?: ReactSpec,
+  conf = {} as ReactSpec,
   _context?: axeCore.ElementContext
 ): Promise<void> {
   React = _React;
   ReactDOM = _ReactDOM;
   timeout = _timeout;
   context = _context;
 
-  const runOnly = (conf || {})['runOnly'];
+  const runOnly = conf['runOnly'];
   if (runOnly) {
     conf['rules'] = axeCore
       .getRules(runOnly)
       .map(rule => ({ ...rule, id: rule.ruleId, enabled: true }));
     conf['disableOtherRules'] = true;
   }
 
-  if (conf) {
+  if (!isEmpty(conf)) {
```

Comment:
Hmm that is a fair point. What about this?

```suggestion
  if (Object.keys(conf).length > 0) {
```

Reply:
Looks good to me!